README.md

```diff
@@ -159,6 +159,7 @@ The following image types are currently available via the `--type` argument:
 | `anaconda-iso`        | An unattended Anaconda installer that installs to the first disk found.               |
 | `raw`                 | Unformatted [raw disk](https://en.wikipedia.org/wiki/Rawdisk).                        |
 | `vhd`                 | [vhd](https://en.wikipedia.org/wiki/VHD_(file_format)) usable in Virtual PC, among others |
+| `gce`                 | GCE input (unformatted raw disk inside a tar.gz file) |
```

Comment:
"GCE input"?  Is "input" a term used in GCE to refer to the image you import?

Reply:
Good catch, this is a bit silly, I made the README line now similar to AMIs, i.e. just point to the upstream google gce docs and assuming people know what GCE means if they needs GCE images. But happy to write more as needed.

Reply:
I think it's fine.  We could expand the acronym so people don't have to click through to see what it means, but I agree that people who need it will know what it means.  Let's merge.  Can always change later.